src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,36 @@
-export const Tabs = () => {};
+import classNames from "classnames";
+export const Tabs = ({tabs, activeTabId, onTabSelected, setOnTabSelected, setActiveTabId}) => {
+    let i = 0;
+    return (
+        <div data-cy="TabsComponent">
+        <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => {
+            
+            return (
+                <li className={classNames({'is-active' : activeTabId === tab.id})} 
+                data-cy="Tab"
+                onClick={ () => {
+                    setOnTabSelected(tab)
```

Comment:
Instead of setting `setOnTabSelected(tab)`, you should call `onTabSelected(tab.id)` if `onTabSelected` is a callback function. This will ensure the parent component handles the tab selection logic.